youtube_to_anki/anki.py

```diff
@@ -7,35 +7,44 @@
 from tempfile import TemporaryDirectory
 from typing import Dict, Iterable
 
+import cv2
 from genanki import Deck, Model, Note, Package
 from pydub import AudioSegment
 
+style = """
+.card {text-align:center;}
+.card {font-family: myfont; font-size: 30px}
+"""
+
 _MODEL = Model(
-    1607392319,
-    "Youtube",
+    1601234219,
+    "YoutubeWithScreenshot",
```

Comment:
There is no need to change these, right?

```suggestion
    1607392319,
    "Youtube",
```

Reply:
reverted it! I thought there might be some conflicts upon importing, but when I tested it the cards looked fine (both .apkg imported into the same anki deck)